testdata/p4tc_samples_outputs/digest.template

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/digest numtables 1
+
+$TC p4template create action/digest/ingress/send_nh actid 1 \
+	param port type dev \
+	param srcMac type macaddr \
+	param dstMac type macaddr
+$TC p4template update action/digest/ingress/send_nh state active
+
+$TC p4template create action/digest/ingress/drop actid 2
+$TC p4template update action/digest/ingress/drop state active
+
+$TC p4template create extern/root/Digest extid 0x05000000 numinstances 1 tc_acl 0x19b6 has_exec_method
+
+$TC p4template create extern_inst/digest/Digest/Ingress_Deparser.digest_inst instid 1 \
```

Comment:
Small detail, `tc_numel` should be specified here

```suggestion
$TC p4template create extern_inst/digest/Digest/Ingress_Deparser.digest_inst instid 1 tc_numel 0 \
```

Reply:
It's because in this case `tc_numel` has a special value (0)
It doesn't really say how many elements digest will have, it serves more as a special flag telling the kernel that digest has some private data

Reply:
ok